devel/docker/Portfile

```diff
@@ -1,10 +1,9 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
+PortGroup           golang 1.0
 
-github.setup        moby moby 17.05.0-ce v
-revision            1
+go.setup            github.com/docker/cli 19.03.6 v
```

Comment:
These days we are preferring that the `revision` line always appear, even when its value is `0`. This makes it easier for other developers to know where in the Portfile a revision should be increased when necessary, and makes it easier to do so programmatically.

Reply:
Fixed, thanks @ryandesign